application/templates/mail/new_request.plush.html

```diff
@@ -1,4 +1,11 @@
+<h4>New Request</h4>
 <p>
-    There is a new request on <a href="<%= uiURL %>"><%= appName %></a>:
+    <a href="<%= postURL %>"><%= postTitle %></a>
```

Comment:
The other messages have this as an `<h4>`. Is it worth being consistent here?

Reply:
Probably. Would it be helpful to have the request description here, or do we want to entice the reader into the app?

Reply:
@fillup What do you think - include the new post's description?